plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,27 @@
+package ru.sbtqa.tag.api.utils;
+
+import java.lang.reflect.Field;
+import ru.sbtqa.tag.api.EndpointEntry;
+import ru.sbtqa.tag.api.exception.RestPluginException;
+
+public class ReflectionUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=81&issues=AWU2KVKnW2AOP0CkD5Fz&open=AWU2KVKnW2AOP0CkD5Fz)

[ISSUE_KEY]: AWU2KVKnW2AOP0CkD5Fz